pghoard/basebackup.py

```diff
@@ -336,6 +336,7 @@ def run_piped_basebackup(self):
         metadata.update({
             "original-file-size": original_input_size,
             "pg-version": self.pg_version_server,
+            "active_backup_mode": self.config["backup_sites"][self.site]["active_backup_mode"],
```

Comment:
```suggestion
            "active_backup_mode": self.config["backup_sites"][self.site]["active_backup_mode"],
            "active_backup_mode": self.site_config["active_backup_mode"],
```
And in other places as well

Reply:
@alexole Ive made the changes as requested.